src/vaadin-custom-field-mixin.html

```diff
@@ -122,13 +131,14 @@
       this.__labelId = `${this.constructor.is}-label-${uniqueId}`;
     }
 
-    /*
-     * @private
-     */
     focus() {
```

Comment:
I'm not sure about this.

This is now without annotation like in [vaadin-list-mixin](https://github.com/vaadin/vaadin-list-mixin/blob/master/vaadin-list-mixin.html#L288) but I noticed that [vaadin-accordion](https://github.com/vaadin/vaadin-accordion/blob/1.2/src/vaadin-accordion.html#L137) uses `@protected` for `focus()`.

Imo semantically this should be public, but I can also understand why we'd like to hide it and as public vs protected doesn't make a difference for the generated TS defs, having this protected would make it a bit more hidden in API docs (and this was previously marked private here).

Reply:
I think we can mark it as `@protected` here as well, for API docs.

Reply:
Done